src/constants/index.js

```diff
@@ -1,5 +1,6 @@
-export const listSenority = ['Junior', 'Mid-Junior', 'Mid', 'Mid-Senior', 'Senior'];
 export const ListEnglish = ['A1', 'A2', 'B1', 'B2', 'C1', 'C2'];
+export const ListCurrencies = ['USD', 'AED', 'AFN', 'ALL', 'AMD', 'ANG'];

```

Comment:
You should create a service for the list of currencies. you can't burn this data for it would change in the future